service-workers/cache-storage/script-tests/cache-storage.js

```diff
@@ -170,8 +170,15 @@ promise_test(function(t) {
 
 promise_test(function(t) {
     var bad_name = 'unpaired\uD800';
-    var converted_name = 'unpaired\uFFFD'; // Don't create cache with this name.
-    return self.caches.has(converted_name)
+    var converted_name = 'unpaired\uFFFD';
+
+    return Promise.all([
+          self.caches.delete(bad_name),
+          self.caches.delete(converted_name)
```

Comment:
Can you just add a clarifying comment to the test?  Looking at it on its own suggests that it should be deleting something, but its really there as a failsafe.  It might be a bit clearer if you just iterated cache.keys() and deleted all cache objects to get a blank slate.